src/createRollupPrepareDeploymentParamsConfig.unit.test.ts

```diff
@@ -16,7 +16,7 @@ function getOverrides({ owner, chainId }: { owner: Address; chainId: bigint }) {
       chainId: Number(chainId),
       arbitrum: {
         InitialChainOwner: owner,
-        InitialArbOSVersion: 30,
+        InitialArbOSVersion: 20,
```

Comment:
This is used to test if overrides work, so we want a version different than the default